ttnn/cpp/ttnn/operations/transformer/sdpa_decode/sdpa_decode.cpp

```diff
@@ -28,14 +28,15 @@ ttnn::Tensor ExecuteScaledDotProductAttentionDecode::invoke(
     const ttnn::Tensor &input_tensor_k,
     const ttnn::Tensor &input_tensor_v,
     const std::vector<uint32_t> cur_pos,
+    const std::optional<const Tensor> cur_pos_tensor,
     std::optional<float> scale,
     const std::optional<MemoryConfig> &memory_config,
     std::optional<SDPAProgramConfig> program_config,
     std::optional<DeviceComputeKernelConfig> compute_kernel_config) {
     auto arch = input_tensor_q.storage_type() == StorageType::DEVICE ? input_tensor_q.device()->arch()
                                                                      : AutoFormat::GetDefaultDevice()->arch();
-    uint32_t max_cur_pos = *std::max_element(cur_pos.begin(), cur_pos.end());
-    uint32_t k_chunk_size = get_chunk_size(max_cur_pos + 1);
+    //uint32_t max_cur_pos = *std::max_element(cur_pos.begin(), cur_pos.end());
+    uint32_t k_chunk_size = 512; //get_chunk_size(max_cur_pos + 1);
```

Comment:
do we always use chunk size 512 now? what's the reason?

Reply:
can't do std::max_element(cur_pos.begin(), cur_pos.end()); now